allauth/templates/account/reauthenticate.html

```diff
@@ -14,7 +14,7 @@
             {{ redirect_field }}
         {% endslot %}
         {% slot actions %}
-            {% element button type="submit" %}
+            {% element button type="submit" tags="primary,login" %}
```

Comment:
I am not sure `login` makes sense here.
```suggestion
            {% element button type="submit" tags="primary,reauthenticate" %}
```